aiorwlock/__init__.py

```diff
@@ -85,11 +85,11 @@ async def acquire_read(self) -> bool:
         self._read_waiters.append(fut)
         try:
             await fut
-            self._r_state += 1
             self._owning.append((me, self._RL))
```

Comment:
Not sure if maybe `self._owning` should also be set immediately, or if it will have no effect on race conditions etc. But, I'm not sure how to get hold of `me` outside the task, so have left it currently.